wordpress-mobile/react-native-aztec
# Integrate Aztec into our Aztec RN wrapper.

### Description:

This PR integrates Aztec into our RN wrapper for it.

Fixes at least the basic part of #25 .  I'll open more atomic issues for the following steps.

### Details:

Creates a subclass of `Aztec.TextView` named `RCTAztecView` with the custom logic for Aztec-in-RN.

Renamed the old `RCTAztecView` to `RCTAztecViewManager`.

Also the text view height should stay in synch with the contents.  Since all cells seem to be sharing the same state object right now, the cell height will affect all cells in the list.

### Preparing to Test:

You will need to have [Carthage installed](https://github.com/Carthage/Carthage#installing-carthage) to test this PR.

You then can use Carthage to retrieve the Aztec dependency.

To do so, open the command line at the root of this project and run the following commands:

```
cd ios
carthage bootstrap --platform iOS
```

Once finished you can move on to testing.

### Testing:

Run the example app and:

- Type in some content.  Add newlines.
- Make sure the content size for the edited cell is mostly right.

### Pending work:

- Make Aztec load the sample content.
- Provide a different state object for each cell in the list so they can have different sizes.
